receiver/k8sclusterreceiver/README.md

```diff
@@ -62,22 +62,25 @@ with detailed sample configurations [here](./testdata/config.yaml).
     k8sclusterreceiver reports container and node cpu metrics in double cpu units instead of integer millicpu units 
     with the feature gate listed below. 
   - Affected Metrics
-    - k8s.container.cpu_request
-    - k8s.container.cpu_limit
-    - k8s.node.allocatable_cpu
+    - Note: These metrics have a current and a legacy name, we list both as pairs (current, legacy) below.
+    - `k8s.container.cpu_request`, `kubernetes.container_cpu_request`
+    - `k8s.container.cpu_limit`, `kubernetes.container_cpu_limit`
+    - `k8s.node.allocatable_cpu`, `kubernetes.node_allocatable_cpu`
```

Comment:
Please discard this change. `kubernetes.*` metrics were never part of the k8sclusterreceiver